### DISCUSSION TITLE
Location of image files in HTML to deploy with golem app

### DISCUSSION THREAD

**Discussion by rezasz969** [ThinkR-open/golem#883] [Q&A]
Hello golem developers,
Thank you for making this great package. I have a problem with HTML file. I put a HTML file into inst/intro. When I run the program, the photos in this file will not be loaded. Can you tell me where I should save the image file?

my app_sever code:
```
app_server <- function(input, output, session) {
  # Your application server logic
  observeEvent("", {
    showModal(modalDialog(
      includeHTML(app_sys("intro","intro_text.html")),
      easyClose = TRUE
    ))
  })
}
```

Thank you!

**Comment by ColinFay** [comment]
Hey, 

This is definitely how you should include it. 

Does `pkgload::load_all();app_sys("intro","intro_text.html")` returns the path?
